common/chain/info_test.go

```diff
@@ -72,7 +73,6 @@ func TestChainInfo(t *testing.T) {
 	c13, err := InfoFromJSON(&c1Buff)
 	require.NoError(t, err)
 	require.NotNil(t, c13)
-	require.Equal(t, c1, c13)
```

Comment:
`require.Equal` if I understand correctly, seems to be comparing the values of the underlying points, which doesn't work for the current bn254 implementation, as the public key is transformed into affine coordinates when being marshaled. So when it is unmarshaled back from JSON, the points may differ. I noticed that in the bls12-381 library, point operations always transform points into affine coordinates.